targets/TARGET_STM/tools/STM32_gen_PeripheralPins.py

```diff
@@ -83,6 +83,7 @@
 MCU_USERNAME= ""
 TIM_MST = ""
 ALTERNATE_DEFINITION = 0
+TargetName = ""
```

Comment:
To be honest, I don't follow any strict coding rules for my personal script :-)
About the issue, it came since the addition of the target name in the PinNames.h file.

I have added it, but script can also create mbed files from only a MCU
So script is now keeping the empty field, up to user to manually update it.

Reply:
>To be honest, I don't follow any strict coding rules for my personal script :-)

Just to be consistent , it would be `TARGET_NAME`

Reply:
@0xc0170 done